guides/how_sprockets_works.md

```diff
@@ -176,131 +181,152 @@ config.assets.precompile = [LOOSE_APP_ASSETS, /(?:\/|\\|\A)application\.(.css|js
-#<script src=”/assets/application.debug-ddbd4593b22ac054471df143715c8ce65ef84938965c7db19d8a322950ec65b6.js”>
+javascript_include_tag 'application'
+#<script src="/assets/application.debug-ddbd4593b22ac054471df143715c8ce65ef84938965c7db19d8a322950ec65b6.js">
 #</script>
 ```
 
-So, when you're having in your code something like `javaScript_include_tag 'application'`, what's going on is that to generate that source attribute of this script tag, Sprockets is going to manifest the object that has a hash like this
+So, when one writes something like `javascript_include_tag 'application'`, what's going on is that Sprockets looks to the manifest to generate the source attribute of this script tag. A sample manifest is shown below that maps the logical path of the file to the filename including the digest:
 
 ```ruby
-{ “application.js” => “application-2e8e9a7888bdbd11e97effec30214a82.js”,
-“application.js” => “application-ae0e5a78gfb231d11e07e00ec30g39f0a.js” }
+{ "application.js" => "application-ae0e5a78gfb231d11e07e00ec30g39f0a.js" }
 ```
 
-
-Inside that only maps the name without the dashes to the name with the dashes and it also contains the opposite way, where you have the daughter's name and you can find either the mtime and the logical path of that file.
+In this example, the logical path of the asset – `application.js` – is mapped to the generated file name of the asset including the digest value. In the example below, which is a reverse mapping, further details of the asset are available including mtime, logical path, and so on.
 
 ```ruby
-{ “application-2e8e9a7888bdbd11e97effec30214a82.js” =>
-     {‘logical-path’ => “application.js”,
-       ‘mtime’ => “2016-06-16T23:09:08-06:00”,
-       ‘Digest’ => “2e8e9a7888bdbd11e97effec30214a82”} }
+{
+  "application-2e8e9a7888bdbd11e97effec30214a82.js" =>
+  {
+    "logical-path" => "application.js",
+    "mtime" => "2016-06-16T23:09:08-06:00",
+    "digest" => "2e8e9a7888bdbd11e97effec30214a82"
+  }
+}
 ```
 
- Another hash is used to expire caches of Sprockets, so you can actually use the same directory and use the assets that you precompiled in the previous deploys. And you can later use this information to expire all the assets that you do not want to be in that folder anymore. There are more things about Sprockets that I'm not going to talk about here, but you can find information, there is Sprockets documentation and also source code. There are mime types, dependency resolvers, transformer suffix, bundle metadata reducer.
+Another hash is used to expire caches of Sprockets, so you can actually use the same directory and use the assets that you precompiled in the previous deploys. Later you can use this information to expire all the assets that you do not want to be in that folder anymore. There are more things about Sprockets that can be found in the Sprockets documentation and source code, including MIME types, dependency resolvers, transformer suffix, bundle metadata reducer.
```

Comment:
I hope this statement is correct. I struggled to interpret what was said in the talk.